File: Algoritmo(C)/Alg_S6_5.c
#include <stdio.h>
int main(){
    //variables
    float peso, exc = 0, multa = 0;
    //input
    printf("Digite o peso total dos peixes: ");
    scanf("%f",&peso);
    if (peso > 50){
        exc = (peso - 50);
        multa = (exc * 4);
    }
    printf("Com o excesso de %.2fKg, o valor da multa e: R$%.2f",exc, multa);
}

File: Algoritmo(Python)/Alg_S6_5.py
#variables
exc = 0
multa = 0
#input
peso = float(input("Digite o peso total: "))
#process
if(peso > 50):
    exc = (peso - 50)
    multa = (exc * 4)
print("Peso: {0:.2f}Kg".format(peso))
print("Excesso: {0:.2f}Kg".format(exc))
print("Multa: R${0:.2f}".format(multa))